Source/control.cpp

```diff
@@ -538,10 +543,10 @@ void ClearPanel()
 	pinfoflag = false;
 }
 
-void DrawPanelBox(const CelOutputBuffer &out, int x, int y, int w, int h, int sx, int sy)
+void DrawPanelBox(const CelOutputBuffer &out, Point sourcePosition, int w, int h, Point targetPosition)

```

Comment:
We might want to use a SDL_rect here since it has width and height also.
I think `CelOutputBuffer` even understands SDL_rect.

Reply:
It understands Point but no SDL_Rect.